processing/src/main/java/org/apache/druid/query/InlineDataSource.java

```diff
@@ -53,14 +63,17 @@
 public class InlineDataSource implements DataSource
 {
   private final Iterable<Object[]> rows;
+  private final Frame frame;
```

Comment:
Lets update the documentation of this class mentioning it can be backed by frames.